src/metabase/driver/mysql.clj

```diff
@@ -77,7 +76,7 @@
   [_driver _feat db]
   (-> db :options :persist-models-enabled))
 
-(doseq [feature [:actions :actions/custom]]
+(doseq [feature [:actions :actions/custom :uploads]]
```

Comment:
the way we do fast inserts for MySQL seems pretty MySQL-specific and I don't want to deal with Maria right now

Reply:
good call, fixed